### PR TITLE
[Docs] Added the docs for Prisma ORM for Node.js

### PR DESCRIPTION
**Pull request by priyanshi-yb**
This PR includes the docs in [Quickstart - build an application](https://docs.yugabyte.com/preview/quick-start/build-apps/nodejs/) and [Drivers and Orms](https://docs.yugabyte.com/preview/drivers-orms/nodejs/) sections for Prisma ORM for Nodejs. This PR depends on the [PR](https://github.com/yugabyte/orm-examples/pull/94) of orm-example of Prisma ORM for Nodejs.

### CODE REVIEW

**Comment by priyanshi-yb** [comment]
@polarweasel , orm-example [PR](https://github.com/yugabyte/orm-examples/pull/94) is merged and now this PR is ready to be reviewed.

**Comment by polarweasel** [comment]
Thanks @aishwarya24 for taking the review for this one!

**Review by aishwarya24 — CHANGES_REQUESTED** [review]

**Comment by aishwarya24** [comment]
@priyanshi-yb , Please sign the CLA.

**Comment by priyanshi-yb** [comment]
@aishwarya24, made the suggested changes.

File: docs/content/preview/quick-start/build-apps/nodejs/ysql-prisma.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+$ npx prisma migrate dev --name first_migration
+```
+ {{< note title="Note" >}}
+
+If you want to use the Prisma CLI without `npx`, install Prisma globally using the following command:
+  ```sh
+  npm install -g prisma
+  ```
+
+{{< /note >}}
```
Comment (polarweasel) [inline docs/content/preview/quick-start/build-apps/nodejs/ysql-prisma.md:135]:
````suggestion
## Apply the migrations 

Create the tables in YugabyteDB by applying the migration for the data models in the file `prisma/schema.prisma`, and generate the Prisma client using the following command:

```sh
$ npx prisma migrate dev --name first_migration
```

 {{< note title="Note" >}}

To use the Prisma CLI without `npx`, install Prisma globally, as follows:

```sh
npm install -g prisma
```

{{< /note >}}

````

File: docs/content/preview/quick-start/build-apps/nodejs/ysql-prisma.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+Start the Node.js API server at <http://localhost:8080> .
+
+```sh
+$ npm start
+```
+If your `PORT` 8080 is already in use, change the port using the following command:
+```sh
+$ export PORT=<new_port>
+```
```
Comment (polarweasel) [inline docs/content/preview/quick-start/build-apps/nodejs/ysql-prisma.md:146]:
````suggestion
Start the Node.js API server at <http://localhost:8080>.

```sh
$ npm start
```

If port 8080 is already in use, change the port using the following command:

```sh
$ export PORT=<new_port>
```

````

**Review by polarweasel — APPROVED** [review]
I'll apply all these suggestions, and then merge

**Pull request merged**